test_runner/fixtures/neon_fixtures.py

```diff
@@ -1438,6 +1498,9 @@ def neon_simple_env(
         pageserver_aux_file_policy=pageserver_aux_file_policy,
         pageserver_default_tenant_config_compaction_algorithm=pageserver_default_tenant_config_compaction_algorithm,
         pageserver_io_buffer_alignment=pageserver_io_buffer_alignment,
+        combination=request._pyfuncitem.callspec.params["combination"]
+        if "combination" in request._pyfuncitem.callspec.params
+        else None,
```

Comment:
We need to move this expression to a separate variable as well.

Reply:
done, thanks